docs: up version and add changelog (#194)
NordicSemiconductor/pc-nrfconnect-npm: Changelog.md

## 2.0.0 - UNRELEASED

### Added

> -   Support for nPM2100 EK.
> -   Virtual Devices feature for offline support.

Reviewer: ```suggestion
-   Support for the nPM2100 Evaluation Kit.
-   Offline support with the virtual device feature.
```

### Changed

> -   The default view is now a Welcome Screen without device specific tabs.
> -   Device specific tabs will be added and removed based on the selected device.

Reviewer: ```suggestion
-   The default view at the application start is now a welcome screen without any device-specific tabs, which are now added and removed based on the selected device.
```

## 1.3.1 - 2024-11-11

> ### Changed

Reviewer: No `nrfutil device` or firmware update in v2.0.0? See line 17 and 18...

Author: I don't think we usually mention the nrfutil device version.
We have done it in specific cases where we had no other changes. But this isn't really a user facing change so we avoid mentioning it.

Reviewer: There are three entries in the changelog about the update/adoption of nrfutil device. Can we be consistent? Can you make a decision whether we keep all these entries and add a new entry or we remove all entries, also from the past updates?

Reviewer 2: As Mia mentioned we tend not to mention nrfutil device update, it should not be there as this is just dependency update. It did appear in the changelogs when we did not have any other updates. Idea to have "Minor fixes and improvements" entry in the changelogs was in general discarded by Bjorn Inge as he wanted to be more specific.
Unfortunately, I see no solution to that but you can force it @greg-fer I think (don't think I can). any decision would work really.

Same for the FW update. Updating dependencies should not be in the changelog, in general.

Reviewer: Deferred decision to Chun on [Teams](https://teams.microsoft.com/l/message/19:meeting_MjNmZjlkM2MtZDk3My00ZGU1LWEzYTAtMDM4NjQ3N2JiMDg3@thread.v2/1742887166032?context=%7B%22contextType%22%3A%22chat%22%7D)
